spec/spec_helper.rb

```diff
@@ -3,13 +3,14 @@
 ENV['RAILS_ENV'] = 'test'
 ENV['environment'] = 'test'
 require 'simplecov'
+require 'valkyrie'
```

Comment:
Either this addition or line 14 is correct, but not the duplication.

Reply:
:+1: